src/oph/ehoks/hoks/schema.clj

```diff
@@ -253,7 +264,8 @@
          "liittyvät tiedot")
     (s/optional-key :osa-aikaisuustieto) s/Int
     (str "Osaamisen hankkimisen osa-aikaisuuden määrä prosentteina (1-100). "
-         "Käytetään työelämäpalautteen työpaikkajakson keston laskemiseen.")
+         "Käytetään työelämäpalautteen työpaikkajakson keston laskemiseen. "
```

Comment:
Tähän voisi myös lisätä kuvaukseen esim. "100 tarkoittaa, että työ on kokoaikaista.  Esimerkiksi 80 tarkoittaa, että työ on osa-aikaista, 80% normaalista kokonaistyöajasta."

Tuo "osa-aikaisuuden määrä" nimittäin kuulostaa siltä, että se voisi kertoa, kuinka paljon normaalista työajasta _puuttuu_.  Mutta se tarkoittaakin, kuinka paljon siitä on jäljellä.  Veikkaan, että useimmat arvaavat merkityksen oikein, mutta selvennys ei liene pahasta.